src/Writer/Extension/PodcastIndex/Feed.php

```diff
@@ -106,6 +110,189 @@ public function setPodcastIndexFunding(array $value): Feed
+            );
+        }
+        if (isset($value['complete']) && ! is_bool($value['complete'])) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: key "complete" of "updateFrequency": must be of type boolean'
+            );
+        }
+        if (isset($value['dtstart']) && ! is_string($value['dtstart'])) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: key "dtstart" of "updateFrequency" must be an ISO8601-formatted string'
+            );
+        }
+        if (isset($value['rrule']) && ! is_string($value['rrule'])) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: key "rrule" of "updateFrequency" must be of type string'
+            );
+        }
+        $this->data['updateFrequency'] = $value;
+        return $this;
+    }
+
+    /**
+     * Add feed person
+     *
+     * @param array<string,mixed> $value [name: str, role: str|null, group: str|null, img: url|null, href: url|null]
+     * @return $this
+     * @throws Writer\Exception\InvalidArgumentException
+     */
+    public function addPodcastIndexPerson(array $value)
+    {
+        if (! isset($value['name'])) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: "person" must be an array containing at least the key "name"'
+            );
+        }
+        if (! is_string($value['name'])) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: key "name" of "person" must be of type string'
+            );
+        }
+        if (isset($value['img']) && ! filter_var($value['img'], FILTER_VALIDATE_URL)) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: key "img" of "person" must be a url, starting with "http://" or "https://"'
+            );
+        }
+        if (isset($value['href']) && ! filter_var($value['href'], FILTER_VALIDATE_URL)) {
+            throw new Writer\Exception\InvalidArgumentException(
+                'invalid parameter: key "href" of "person" must be a url, starting with "http://" or "https://"'
+            );
+        }
+        if (! isset($this->data['persons'])) {
+            $this->data['persons'] = [];
+        }
+
+        /** @var array<array<string, mixed>> $this->data['persons'] */
+        $this->data['persons'][] = $value;
+        return $this;
+    }
+
+    /**
+     * Set a new array of persons.
+     * If no argument is passed, it will just remove all existing persons.
+     *
+     * @param null|array<array> $values
+     * @return $this
+     * @throws Writer\Exception\InvalidArgumentException
+     */
+    public function setPodcastIndexPersons(?array $values = null)
+    {
+        // delete existing persons before setting new ones
+        $this->data['persons'] = null;
```

Comment:
With `$values` always an array

```suggestion
        $this->data['persons'] = [];
```

And you can drop the subsequent conditional